File: Calc/app/src/main/java/com/qwer/calc/MainActivity.java
package com.qwer.calc;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView result;
    EditText n1, n2;
    Button bt1, bt2, bt3, bt4;

    float res;
    int num1, num2;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        result = (TextView)findViewById(R.id.result);
        n1 = (EditText)findViewById(R.id.number1);
        n2 = (EditText)findViewById(R.id.number2);
        bt1 = (Button)findViewById(R.id.add);
        bt2 = (Button)findViewById(R.id.substract);
        bt3 = (Button)findViewById(R.id.multiply);
        bt4 = (Button)findViewById(R.id.divide);

        bt1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                num1 = Integer.parseInt(n1.getText().toString());
                num2 = Integer.parseInt(n2.getText().toString());
                res = num1 + num2;
                result.setText(String.valueOf(res));
            }
        });

        bt2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                num1 = Integer.parseInt(n1.getText().toString());
                num2 = Integer.parseInt(n2.getText().toString());
                res = num1 - num2;
                result.setText(String.valueOf(res));
            }
        });

        bt3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                num1 = Integer.parseInt(n1.getText().toString());
                num2 = Integer.parseInt(n2.getText().toString());
                res = num1 * num2;
                result.setText(String.valueOf(res));
            }
        });

        bt4.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                num1 = Integer.parseInt(n1.getText().toString());
                num2 = Integer.parseInt(n2.getText().toString());
                res = num1 / num2;
                result.setText(String.valueOf(res));
            }
        });


    }
}

File: Fragments/app/src/main/java/com/qwer/fragments/MainActivity.kt
package com.qwer.fragments

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.qwer.fragments.FirstFragment
import com.qwer.fragments.R
import com.qwer.fragments.SecondFragment

class MainActivity : AppCompatActivity(), MyListener {

    lateinit var fragmentFirst: FirstFragment
    lateinit var fragmentSecond: SecondFragment

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        fragmentFirst = FirstFragment.newInstance("", "")
        fragmentSecond = SecondFragment.newInstance("", "")

        supportFragmentManager.beginTransaction()
                .add(R.id.firstContainer, fragmentFirst)
                .commit()

        supportFragmentManager.beginTransaction()
                .add(R.id.secondContainer, fragmentSecond)
                .commit()

    }

    override fun onFragmentButtonClicked() {
        fragmentFirst.plus()
    }
}

interface MyListener {
    fun onFragmentButtonClicked()
}